### ISSUE TITLE
Add Multiple Particle Types

### ISSUE THREAD

**Issue by schmoelder** [fau-advanced-separations/CADET-Process#4]
In **CADET** [multiple particle types](https://cadet.github.io/master/modelling/unit_operations/general_rate_model.html#multiple-particle-types) are supported. Currently, this feature is missing in **CADET-Process**.

See also: https://github.com/cadet/CADET-Specification/issues/4

## To do
- [ ] Update `create_LWE` method to allow for multiple particle types

**Comment by hannahlanzrath** [comment]
When this is implemented, in the upcoming create_LWE() method, multiple particle types have to be included (based on [createLWE](https://github.com/modsim/CADET/blob/master/src/tools/createLWE.cpp) in CADET-core).

**Comment by nbarry03** [comment]
Is this something that still needs to be implemented? I'm looking at adding this as I need to model particle size distributions myself.

**Comment by schmoelder** [comment]
Yes, but I have a particular design in mind that still needs some "brewing", if you know what I mean. In short, I want to introduce another abstraction layer:

`UnitOperation` can have multiple `ParticleType`s which each manage their own `BindingModel`. However, this also means, we need to change where many of the parameter live. However, I believe, it's a much more modular design. 

We can have a chat about it if you want, ideally during our next [Office Hours](https://forum.cadet-web.de/t/introducing-office-hours/518).

**Comment by nbarry03** [comment]
That sounds good, I have something hacked together temporarily which will work in the meantime. I'll plan on joining the next office hours session.
